dev/com.ibm.ws.jaxws.2.3.clientcontainer/src/com/ibm/ws/jaxws/bus/LibertyApplicationBusFactory.java

```diff
@@ -115,6 +133,16 @@ public LibertyApplicationBus createBus(Map<Class<?>, Object> e, Map<String, Obje
             }
 
             bus.initialize();
+
+            // Always register LibertyLoggingIn(Out)Interceptor Pretty print the SOAP Messages
+            final LibertyLoggingInInterceptor in = new LibertyLoggingInInterceptor();
```

Comment:
Add a TODO to remove the LoggingInterceptors from the client and server, and switch to using the LoggingFeature